src/utils/appinsights.ts

```diff
@@ -54,6 +57,16 @@ export function initAppInsights(
     appInsights.defaultClient.context.keys.cloudRole
   ] = getValueFromPackageJson("name");
 
+  if (config.httpAgent !== undefined) {
+    // tslint:disable-next-line: no-object-mutation
+    appInsights.defaultClient.config.httpAgent = config.httpAgent;
+  }
+
+  if (config.httpsAgent !== undefined) {
+    // tslint:disable-next-line: no-object-mutation
+    appInsights.defaultClient.config.httpAgent = config.httpsAgent;
```

Comment:
isn't this https ? I've applied an hotfix here:

https://github.com/pagopa/io-backend/commit/aaed7fffe26cf1d4b4c61b8d4938a4f7af9e43b1

@cloudify

Reply:
yes my mistake